src/Modules/FudgeTools.psm1

```diff
@@ -1714,6 +1715,32 @@ function Get-ChocolateyLatestVersion
     return 'latest'
 }
 
+function Update-EnvironmentPath
```

Comment:
this looks quite good, and a new creative work for sure.

some new tests would be useful, but not as important as tests for Update-EnvironmentVariables

I think this one can be manually tested.  Ill try to get to it today.